src/typings/geojson.d.ts

```diff
@@ -0,0 +1,147 @@
+export type GeoJSONGeometry =
```

Comment:
There is already a package for GeoJSON types at https://www.npmjs.com/package/@types/geojson. It would be better we use those and maybe extend them if we need to add STAC extras, rather than recreating a whole types package.